src/middleware/index.js

```diff
@@ -43,14 +43,17 @@ const analyticsMiddleware = store => next => action => {
     let details = {
         pathname: window.location.pathname,
     }
+    if (action.type === 'ADD_ACCESS_KEY') {
+        details['appTitle'] = action.meta.data.title
```

Comment:
@kcole16 looks like long-term we should log contract ID here, generally we should stop using titles supplied by user.